### PR TITLE
Updating sha fields via JQ

### PR DESCRIPTION
**Pull request by adrian-kong**

### CODE REVIEW

File: link_release.sh [hunk 1]
```diff
@@ -50,9 +54,9 @@ NEW_RELEASE=$(jq \
   | .base_dir.macos=$dir_mac
   | .base_dir.windows=$dir_win
 
-  | .sha256.linux=sha_linux
-  | .sha256.macos=sha_macos
-  | .sha256.windows=sha_windows
+  | .sha256.linux=$sha_linux
```
Comment (adrian-kong) [inline link_release.sh:57]: think you were just missing the $ signs here

File: link_release.sh [hunk 2]
```diff
@@ -9,20 +9,20 @@ sha_macos=""
 sha_windows=""
```
Comment (adrian-kong) [inline link_release.sh:9]: can remove these I believe, would be "" on default if I'm not mistaken

File: link_release.sh [hunk 3]
```diff
@@ -34,6 +34,10 @@ NEW_RELEASE=$(jq \
   --arg dir_mac "${DIR_MAC}" \
   --arg dir_win "${DIR_WIN}" \
   \
+  --arg sha_linux "${sha_linux}" \
+  --arg sha_macos "${sha_macos}" \
+  --arg sha_windows "${sha_windows}" \
```
Comment (adrian-kong) [inline link_release.sh:39]: pipe in as argument just for clarity, don't think these --arg are essential but are nice

File: link_release.sh [hunk 4]
```diff
@@ -9,20 +9,20 @@ sha_macos=""
...
-  echo hello
   sha_macos=shasum -a 256 "$DL_MACOS"
+  echo sha_macos
 fi
 
 if test -f "${DL_WIN}";
 then 
-  echo hello
   sha_windows=shasum -a 256 "$DL_WIN"
+  echo sha_windows
```
Comment (adrian-kong) [inline link_release.sh:25]: just echo these as debugging,
think to clean this up we can use an array 

maybe something like:
```sh
DL_FILES=($DL_LINUX $DL_MACOS $DL_WIN)
for i in "${DL_FILES[@]}"
do
    declare "sha_$i=$(shasum -a 256 $i)"
done
```

we can probably pass the args without preventing file not found with directly
```sh
--arg sha_linux "$(sha256sum $DIR_LINUX)"
```
or something like that to clean up

**Review by achubbic-snav — APPROVED** [review]
TODO: remove debug echos

**Pull request merged**